Equilobe.DailyReport.Models/Jira/Entries.cs

```diff
@@ -1,40 +1,72 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Xml.Serialization;
+using System.Runtime.Serialization;
 
 namespace Equilobe.DailyReport.Models.Jira
 {
-    [Serializable]
+    [DataContract]
     public class Entry
     {
-        [XmlElement("comment")]
+        [DataMember(Name = "comment")]
         public string Comment { get; set; }
 
-        [XmlElement("timeSpent")]
+        [DataMember(Name = "timeSpent")]
         public int TimeSpent { get; set; }
 
-        [XmlElement("author")]
+        [DataMember(Name = "author")]
         public string Author { get; set; }
 
-        [XmlElement("authorFullName")]
+        [DataMember(Name = "authorFullName")]
         public string AuthorFullName { get; set; }
 
-        [XmlElement("created")]
-        public DateTime Created { get; set; }
+        [DataMember(Name = "created")]
+        public string Created { get; set; }
 
-        [XmlElement("startDate")]
-        public DateTime StartDate { get; set; }
+        [DataMember(Name = "startDate")]
+        public string StartDate { get; set; }
 
-        [XmlElement("updateAuthor")]
+        [DataMember(Name = "updateAuthor")]
         public string UpdateAuthor { get; set; }
 
-        [XmlElement("updateAuthorFullName")]
+        [DataMember(Name = "updateAuthorFullName")]
         public string UpdateAuthorFullName { get; set; }
 
-        [XmlElement("updated")]
-        public DateTime Updated { get; set; }
+        [DataMember(Name = "updated")]
+        public string Updated { get; set; }
+
+        public DateTime CreatedAt
+        {
+            get
+            {
+                return Convert.ToDateTime(Created);
+            }
+            set
+            {
```

Comment:
The properties are used throughout the project, and if I remove the set, even though it's empty, it would mean that the property is read only. Having an empty set, means that I can set the property's value outside the class.

Reply:
Is this set outside this? For me it seems that we only need the getter.

Reply:
Yes, it is used in IssueAdapter